src/controllers/commentController.js

```diff
@@ -0,0 +1,97 @@
+import commentServices from "../services/commentServices";
+import commentValidator from "../validation/commentValidation";
+import Util from "../utilities/util";
+import db from "../models";
+
+const util = new Util();
+
+const commentController = {
```

Comment:
Is there a specific reason why you used constants instead of classes and static methods?